tensorboard/components/experimental/plugin_lib/BUILD

```diff
@@ -1,32 +1,43 @@
 package(default_visibility = ["//tensorboard:internal"])
 
 load("//tensorboard/defs:web.bzl", "tf_web_library")
+load("//tensorboard/defs:defs.bzl", "tf_ts_library", "tf_js_binary")
 
 licenses(["notice"])  # Apache 2.0
 
+# Serve for tests only.
 tf_web_library(
-    name = "guest_internals",
+    name = "test_lib",
     srcs = [
+        "runs.ts",
+        "message.ts",
         "plugin-guest.ts",
+        "tf-plugin-lib.ts",
     ],
     path = "/tf-plugin-lib",
-    deps = [
-        "//tensorboard/components/experimental/plugin_util:message",
-    ],
+    visibility = ["//visibility:public"],
 )
 
-# TODO(psybuzz): figure out how this tf_web_library can be used to create
-# maybe a NPM package.
-tf_web_library(
-    name = "plugin_lib",
+tf_ts_library(
+    name = "ts_plugin_lib",
     srcs = [
+        "message.ts",
+        "plugin-guest.ts",
         "runs.ts",
-        "tf-plugin-lib.html",
+        "tf-plugin-lib.ts",
     ],
-    path = "/tf-plugin-lib",
-    visibility = ["//visibility:public"],
+)
+
+# To update the plugin lib for clients relying on a source set of ES6 Modules:
+# > bazel build tensorboard/components/experimental/plugin_lib:tb_plugin_lib
+# Then copy this folder
+# cp -r ./bazel-bin/tensorboard/components/experimental/plugin_lib/tb_plugin_lib.es6/tensorboard/components/experimental/plugin_lib/ tensorboard/components/experimental/plugin_lib/dist
+# into dist/
+tf_js_binary(
+    name = "tb_plugin_lib",
```

Comment:
if you are not going to commit the output of the binary, you probably don't need to have this target.

Instead, `bazel build tensorboard/components/experimental/plugin_lib:ts_plugin_lib` should be sufficient.

Reply:
Actually, we need this, as the ts_plugin_lib only produces ES5 output.  Added a comment